guides/schema/lazy_execution.md

```diff
@@ -4,7 +4,8 @@ doc_stub: false
 search: true
 title: Lazy Execution
 section: Schema
-desc: Resolve functions can return "unfinished" results. GraphQL will defer finishing them until other fields have been resolved.
+desc: Resolve functions can return "unfinished" results that are deferred for batch resolution.
```

Comment:
Just a little re-write to make the description fit into one line on the menu :smiley_cat: